### ISSUE TITLE
Validation loss is lower than training loss in MNIST tutorial

### ISSUE THREAD

**Issue by Ajay1590** [skorch-dev/skorch#722]
Hi! I've been using skorch for some of my own research projects, and i noticed that during the training process, the validation loss was always lower than the training loss. This is the case whether i specify the parameter:

train_split = predefined_split(valid_ds) 

where i have split the data myself, or leave the train_split to the default. I noticed that in the MNIST example, for both the fully connected and  convolutional networks, the training loss is higher than the validation loss at each epoch. To my knowledge, this should not be possible with proper data shuffling - is there a reason that skorch classifiers seem to exhibit this trend?

**Comment by BenjaminBossan** [comment]
Two possible reasons:

1. The training loss is averaged over each batch _while_ the net is training. The validation loss is calculated _after_ the end of the epoch, thus the net is already more trained at that point.
2. Just be random chance.

**Comment by BenjaminBossan** [comment]
@Ajay1590 I will close this issue for now. If you find that none of the answers fit, please report and re-open the issue.
